clab/config/template.go

```diff
@@ -40,6 +40,10 @@ func LoadTemplates(tmpl *template.Template, role string) error {
 		fn := filepath.Join(p, fmt.Sprintf("*__%s.tmpl", role))
 		_, err := tmpl.ParseGlob(fn)
 		if err != nil {
+			if strings.Contains(err.Error(), "pattern matches no file") {
+				log.Debug(err)
+				return nil
+			}
```

Comment:
Hi @kellerza 
can you remind me what is the idea here? Why we want to let this error slip?

PS> If this is valid reason, lets look if ParseGlob uses a certain error for it, so we can use `errors.Is()` instead of checking on string value of an error

Reply:
> can you remind me what is the idea here? Why we want to let this error slip?

We parse template roles using wildcards (i.e. `*__ce.tmpl` and `*__srl.tmpl`). 
Certain roles might not have templates defined (i.e. `bgp__srl.tmpl`, but there are no `bgp__ce.tmpl` in the directory)

Initially we failed on these errors and empty template files were required.

A feature was added to not require such empty templates [here](https://github.com/srl-labs/containerlab/commit/c008463ae3b201a62449ceb079246a032b191628#diff-ac45aaa99ba2c73d9e9799f88ff0d26d0cde7d54d0fe690e60d2c8f21ebdf90dR78). Unfortunately this also introduced the regressions that you can have a template file present that is unable to load. Example is an invalid template function name (these functions are resolved on loading templates, not Execution).

**In summary this PR fails with Invalid templates and hides unavailable templates**

> PS> If this is valid reason, lets look if ParseGlob uses a certain error for it, so we can use errors.Is() instead of checking on string value of an error

[text/template] only exposes an [evaluation error](https://pkg.go.dev/text/template#ExecError) and the go source is unavailable at present https://cs.opensource.google/go/go